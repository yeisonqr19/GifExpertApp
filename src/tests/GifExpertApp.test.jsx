import React from "react";
import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import GifExpertApp from "../GifExpertApp";

describe("Pruebas para el Componente Principal <GifExpertApp/>", () => {
  test("Debe renderizar el componente correctamente y hacer un snapshot", () => {

    const wrapper = shallow(<GifExpertApp />);
    expect(wrapper).toMatchSnapshot();

  });  
});
