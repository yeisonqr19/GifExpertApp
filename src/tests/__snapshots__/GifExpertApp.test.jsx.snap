// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas para el Componente Principal <GifExpertApp/> Debe renderizar el componente correctamente y hacer un snapshot 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="One piece"
      key="One piece"
    />
  </ol>
</Fragment>
`;
